projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('AuthService', () => {
       '__access_token_from_popup__'
     );
 
-    locationSpy = jasmine.createSpyObj('Location', ['path']);
+    locationSpy = jasmine.createSpyObj('Location', ['path', 'replaceState']);
```

Comment:
Do we still need this replaceState? I am not sure what its doing.

Reply:
I've removed it now. I think it's an overhang from when I started writing the tests. The Abstract Navigator uses `replaceState` if there is no router, and the navigator is called by `handleRedirectCallback`. However, I later updated the test so that it completely sets up the testbed anew, which contains a router, so `replaceState` is never called and thus no longer needs to be set up in the location spy. Thanks for calling out!